pkg/get/tools.go

```diff
@@ -1168,5 +1168,50 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				{{.Name}}_{{.Version}}_{{$osString}}_{{$arch}}{{$ext}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:   "argoproj-labs",
+			Repo:    "argocd-autopilot",
+			Name:    "argocd-autopilot",
+			Version: "0.2.1",
+			URLTemplate: `
+			{{$arch := ""}}
+			{{- if eq .Arch "x86_64" -}}
+			{{$arch = "amd64"}}
+			{{- else if eq .Arch "aarch64" -}}
+			{{$arch = "arm64"}}
+			{{- end -}}
+
+			{{$osString := ""}}
+			{{ if HasPrefix .OS "ming" -}}
+			{{$osString = "windows"}}
+			{{- else if eq .OS "linux" -}}
+			{{$osString = "linux"}}
+			{{- else if eq .OS "darwin" -}}
+			{{$osString = "darwin"}}
+			{{- end -}}
+			{{$ext := ".tar.gz"}}
+			https://github.com/{{.Owner}}/{{.Repo}}/releases/download/v{{.Version}}/{{.Name}}-{{$osString}}-{{$arch}}{{$ext}}
+			`,
+			BinaryTemplate: `
```

Comment:
What happens if you remove the `BinaryTemplate`, does it still work?

Reply:
No, because then it uses the tool Name but inside of the tar the binary ist with some arch infos

See picture:
![image](https://user-images.githubusercontent.com/38325136/121651264-b677a200-ca9a-11eb-8a94-150184c89c34.png)

I think about to do a PR on argocd-autopilot to change this to just the cli name. But you never know if this could happen on another tool too

Reply:
Thanks for the comment, and especially for the concrete example.

cc @Waterdrips